.github/workflows/conformance-ginkgo.yaml

```diff
@@ -265,6 +265,7 @@ jobs:
       - name: Install helm
         shell: bash
         run: |
+          # renovate: datasource=github-releases depName=helm/helm
           HELM_VERSION=3.7.0
           wget "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz"
```

Comment:
do we need to put v prefix + and remove v in wget command ?

Reply:
I think you're right!